hooks/event-conversion-utils.ts

```diff
@@ -59,14 +54,63 @@ function getEventsQuery (apiKey: string, days: number, view: string, filters: st
   })
 }
 
+/**
+ * Check whether an event happens within a predetermined number of days before
+ * or after today.
+ * If the "days" parameter is positive, the event must happen in the future,
+ * between today and the given number of days after today.
+ * If the "days" parameter is negative, the event must happen in the past,
+ * between today and the given number of days before today.
+ * @param event The event to check.
+ * @param days The number of days before and after today.
+ * @returns Whether the event happens within the specified range.
+ */
+function isEventInDateRange (
```

Comment:
I really like this @eddybrando — this _does_ feel better than struggling w/ Airtable filters